### PR TITLE
Image sensor cleanup

### PR DESCRIPTION
**Pull request by jaredweiss**
Several updates to the ImageSensor to make it easy to work with within the Networks API of nupic. 

@scottpurdy Please review

### CODE REVIEW

File: nupicvision/regions/ImageSensor.py [hunk 1]
```diff
@@ -2569,10 +2570,10 @@ def getSpec(cls):
           accessMode='ReadWrite'
         ),
         outputImage=dict(
-          description="""Serialized version of the current output image(s). If depth > 1,
+          description="""YAML serialized version of the current output image(s). If depth > 1,
             multiple serialized images will be returned in a list. To deserialize:
             from nupicvision.image import deserializeImage
-            outputImage = deserializeImage(sensor.getParameter('outputImage'))""",
+            outputImage = deserializeImage(yaml.load(sensor.getParameter('outputImage')))""",
```
Comment (scottpurdy) [inline nupicvision/regions/ImageSensor.py]: line too long

File: nupicvision/regions/ImageSensor.py [hunk 2]
```diff
@@ -2569,10 +2570,10 @@ def getSpec(cls):
           accessMode='ReadWrite'
         ),
         outputImage=dict(
-          description="""Serialized version of the current output image(s). If depth > 1,
+          description="""YAML serialized version of the current output image(s). If depth > 1,
```
Comment (scottpurdy) [inline nupicvision/regions/ImageSensor.py]: line too long here and elsewhere

**Comment by scottpurdy** [comment]
:+1: after line lengths are fixed

**Comment by jaredweiss** [comment]
ping @scottpurdy

**Pull request merged**